src/util.ts

```diff
@@ -163,3 +164,38 @@ export function stringifyRequest(
       .join('!')
   )
 }
+
+export function genMatchResource(
+  context: LoaderContext<VueLoaderOptions>,
+  resourcePath: string,
+  resourceQuery?: string,
+  lang?: string,
+  additionalLoaders?: string[]
```

Comment:
I don't see any usage of this parameter. What's it reserved for?

Reply:
This is not related. I will remove this.